subsystems/viamserver/viamserver.go

```diff
@@ -168,9 +181,24 @@ func (s *viamServer) Start(ctx context.Context) error {
 	select {
 	case matches := <-c:
 		s.checkURL = matches[1]
-		s.checkURLAlt = strings.Replace(matches[2], "0.0.0.0", "localhost", 1)
-		s.logger.Infof("viam-server restart allowed check URLs: %s %s", s.checkURL, s.checkURLAlt)
+		s.checkURLAlt = strings.Replace(matches[2], "0.0.0.0", "127.0.0.1", 1)
```

Comment:
I changed this to `127.0.0.1` so checks to `checkURLAlt` would work on Windows (`localhost` does not exist on Windows) as well as Linux. Restart allowed checks _could_ already work with the `checkURL` (this was something like `https://vader-main.ydbecmp2sz.local.viam.cloud:8080`).